File: backend/src/main/java/ru/focsit/backend/dto/SignInRequest.java
package ru.focsit.backend.dto;

import jakarta.validation.constraints.NotBlank;
import lombok.Data;

@Data
public class SignInRequest {

    @NotBlank(message = "Имя пользователя не может быть пустыми")
    private String username;

    @NotBlank(message = "Пароль не может быть пустыми")
    private String password;
}

File: mobile/app/src/main/java/ru/focsit/mobile/data/auth/SignInRequest.kt
package ru.focsit.mobile.data.auth

/**
 * Запрос на авторизацию пользователя в системе.
 * Этот класс используется для отправки данных пользователя при входе в систему.
 *
 * @property username Имя пользователя, которое будет использоваться для входа в систему.
 * @property password Пароль пользователя для доступа к его учетной записи.
 */
data class SignInRequest(
    val username: String,
    val password: String
)